### ISSUE TITLE
workflow and system tag

### ISSUE THREAD

**Issue by ArtificialOwl** [ArtificialOwl/workflow_photo_organizer#1]
With the current configuration, the event will be called even when uploading a file at the root of the filesystem

![Selection_027](https://user-images.githubusercontent.com/1038617/161744649-96481ae9-54eb-4608-b2f5-a4549f036ef3.png)

Is it a normal behavior ?

**Comment by blizzz** [comment]
Yes, for you still need to get the matching Flows as per getFlows() and act upon it. 

Have a look at https://github.com/Rello/analytics/blob/master/lib/Flow/FlowOperation.php#L95-L120 or  https://github.com/nextcloud/flow_notifications/blob/master/lib/Flow/Operation.php#L111-L159 for existing code.
